Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
 def customiseDataRun2Common_25ns(process):
     process = customiseDataRun2Common(process)
 
+    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
+    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
```

Comment:
Hi Salavat,

please use
runTheMatrix.py -l 134.802
this is one of the 2015C relvals




On 9/25/15 6:50 AM, Salavat Abdullin wrote:
> In Configuration/DataProcessing/python/RecoTLR.py
> <https://github.com/cms-sw/cmssw/pull/11481#discussion_r40421749>:
> 
>> @@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
>>  def customiseDataRun2Common_25ns(process):
>>      process = customiseDataRun2Common(process)
>>  
>> +    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
>> +    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
> 
> Slava, could you just give your option (example)?
> I have only complied and run 'runTheMatrix --useInput all &'
> which obviously don't act on 25ns data.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11481/files#r40421749>.
>